draft-ietf-quic-transport.md

```diff
@@ -2675,201 +2676,291 @@ created, it MUST close the connection with error code STREAM_STATE_ERROR.
 Stream ID 0 (0x0) is reserved for the cryptographic handshake.  Stream 0 MUST
 NOT be used for application data, and is the first client-initiated stream.
 
-A QUIC endpoint MUST NOT reuse a Stream ID.  Streams MUST be created
-in sequential order.  Open streams can be used in any order.  Streams
-that are used out of order result in lower-numbered streams in the
-same direction being counted as open.
+A QUIC endpoint MUST NOT reuse a Stream ID.  Open streams can be used in any
```

Comment:
I forgot that I did that.  Since it's unenforceable, can't we just concede that open-in-order isn't strictly an interoperability requirement?  I could restore text with weasel words, but I've been convinced that strictly legislating here isn't constructive.

Reply:
Fair enough, make sure to find and close the issue this was discussed in then.